packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1661,6 +1689,30 @@ export class Vector3 {
         return s;
     }
 
+    /**
+     * Get rotation needed to to rotate from one Vector3 onto another Vector3
+     * @param fromVector the starting vector
+     * @param toVector the ending vector
+     * @returns the rotation needed
+     */
+    public static RotationFromOnto(fromVector: DeepImmutable<Vector3>, toVector: DeepImmutable<Vector3>) {

```

Comment:
We should remove the functions here. It is adding extra code and I would prefer to be mindful with package size. Let s keep it only in the quaternion class ?

Reply:
Reasonable, do it tomorrow.

Reply:
no rush it won t be merged before Monday anyway ;-)

Reply:
Done